docs/src/examples/sensitivity-analysis-ridge.jl

```diff
@@ -0,0 +1,132 @@
+# # Sensitivity Analysis of Ridge Regression
+
+#md # [![](https://img.shields.io/badge/show-github-579ACA.svg)](@__REPO_ROOT_URL__/docs/src/examples/sensitivity-analysis-ridge.jl)
+#md # [![](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/generated/sensitivity-analysis-ridge.ipynb)
+
+# This example illustrates the sensitivity analysis of data points in a
+# [Ridge Regression](https://en.wikipedia.org/wiki/Ridge_regression) problem.
+# The general form of the problem is given below:
+
+# ```math
+# \begin{split}
+# \begin{array} {ll}
+# \mbox{minimize} & \sum_{i=1}^{N} (y_{i} - w x_{i} - b)^2 + \alpha (w^2 + b^2) \\
+# \end{array}
+# \end{split}
+# ```
+# where
+# - `w`, `b` are slope and intercept of the regressing line
+# - `x`, `y` are the N data points
+# - `α` is the regularization constant
+#
+# which is equivalent to:
+# ```math
+# \begin{split}
+# \begin{array} {ll}
+# \mbox{minimize} & e^{\top}e + \alpha (w^2 + b^2) \\
+# \mbox{s.t.} & e_{i} = y_{i} - w x_{i} - b \quad \quad i=1..N  \\
+# \end{array}
+# \end{split}
+# ```
+
+
+
+# This tutorial uses the following packages
+
+using JuMP
+import DiffOpt
+import Random
+import OSQP
+import Plots
+import LinearAlgebra: normalize!, dot
+
+# ## Define and solve the problem
+
+# Construct a set of noisy (guassian) data points around a line.
+
+Random.seed!(42)
+
+N = 100
+
+w = 2 * abs(randn())
+b = rand()
+X = randn(N)
+Y = w * X .+ b + 0.8 * randn(N);
+
+# The helper method `fitRidge` defines and solves the corresponding model.
```

Comment:
`fit_ridge` would be more idiomatic.
Also some more comments maybe between the code parts?